gemrb/GUIScripts/iwd2/MessageWindow.py

```diff
@@ -71,29 +70,27 @@ def UpdateControlStatus():
 	if Override:
 		Expand = GS_LARGEDIALOG
 
-	GemRB.LoadWindowPack(GUICommon.GetWindowPack())
-	hideflag = IsGameGUIHidden()
+	MessageWindow = GemRB.LoadWindow(0, GUICommon.GetWindowPack())
```

Comment:
I dont recall for certain, but I think IWD2 doesn't have a separate OptionWindow. Seems like its part of the MessageWindow.